packages/client/subscriptions/TeamSubscription.ts

```diff
@@ -207,7 +203,6 @@ const updateHandlers = {
   RemoveAgendaItemPayload: removeAgendaItemUpdater,
   UpdateAgendaItemPayload: updateAgendaItemUpdater,
   AcceptTeamInvitationPayload: acceptTeamInvitationTeamUpdater,
-  AddReflectTemplatePayload: addReflectTemplateTeamUpdater,
```

Comment:
-1 Sorry for not tagging earlier. In my head it belonged to my other comment. Please re-add this updated updater

Reply:
Feel free to merge afterwards